s2d-oms-mgmt-solution/azuredeploy.json

```diff
@@ -0,0 +1,1847 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsLogAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of existing OMS Log Analytics workspace."
+      }
+    },
+    "omsLogAnalyticsWorkspaceLocation": {
+      "type": "string",
+      "allowedValues": [
```

Comment:
@gatneil  I've updated the parameters file and removed the API version. Seems I've forgot to remove it when I've made the change to the template. I am still getting error on build and still the error is general without any explanation. On the old PRs I can see now what where the errors. May be something is not working correctly as it does not show the full error right away and you have to wait some time. The location of the Resource Group could be different than the actual location of the resource (workspace). By specifying the allowed locations as parameter I am making sure there will be no errors in deploying it in the wrong region where workspace is not available. I do not think I should change that.